Trying to consume a HTTPs endpoint not find sun/security/ssl/HandshakeStateManager

## Environment Details
* GlassFish Version (and build number): 
5.1.0 build default-private from http://mirrors.xmission.com/eclipse/glassfish/glassfish-5.1.0.zip
* JDK version: 
openjdk version "1.8.0_121"
OpenJDK Runtime Environment (build 1.8.0_121-b13)
OpenJDK 64-Bit Server VM (build 25.121-b13, mixed mode)
* OS: Centos:7

----------

## Problem Description
Trying to consume a restful HTTPS endpoint from the application to an external service in AWS, I got this:

Caused by: java.lang.NoClassDefFoundError: sun/security/ssl/HandshakeStateManager
	at sun.security.ssl.Handshaker.init(Handshaker.java:282)
	at sun.security.ssl.Handshaker.<init>(Handshaker.java:242)
	at sun.security.ssl.ClientHandshaker.<init>(ClientHandshaker.java:160)
	at sun.security.ssl.SSLSocketImpl.initHandshaker(SSLSocketImpl.java:1329)
	at sun.security.ssl.SSLSocketImpl.doneConnect(SSLSocketImpl.java:690)
	at sun.security.ssl.SSLSocketImpl.<init>(SSLSocketImpl.java:569)
	at sun.security.ssl.SSLSocketFactoryImpl.createSocket(SSLSocketFactoryImpl.java:110)
	at org.apache.http.conn.ssl.SSLConnectionSocketFactory.createLayeredSocket(SSLConnectionSocketFactory.java:365)
	at org.apache.http.conn.ssl.SSLConnectionSocketFactory.connectSocket(SSLConnectionSocketFactory.java:355)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:373)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:56)

## Steps to reproduce
Call an HTTPS endpoint from the application

## Impact of Issue
I am trying to deploy a Glassfish docker image using airhacks/glassfish:5.1.0 in AWS ECS

--- a/nucleus/common/common-util/src/main/java/com/sun/logging/LogCleanerUtil.java
+++ b/nucleus/common/common-util/src/main/java/com/sun/logging/LogCleanerUtil.java
@@ -66,6 +66,9 @@ public final class LogCleanerUtil {
 
 
     public static String neutralizeForLog(String message){
+        if (message == null) {
+            return null;
+        }
         StringBuilder sb = new StringBuilder();
         for(int offset  = 0; offset < message.length(); ){
             final int point = message.codePointAt(offset);

